src/webapp/logic/sheetBuilder.ts

```diff
@@ -680,6 +680,21 @@ export class SheetBuilder {
         validationSheet.cell(1, 1, 1, columnId, true).formula(`_${element.id}`).style(baseStyle);
     }
 
+    private getGroupingKey(item: TranslatableItem) {
+        switch (item.type) {
+            case "categoryOptionCombos":
+                return `${item.type}_${item.categoryCombo?.id || "unknown"}`;
+
+            case "dataElements":
+                if (item.categoryCombo?.id) {
+                    return `${item.type}_combo_${item.categoryCombo.id}`;
+                }
+                return item.type;
```

Comment:
Probably not important, but It's a bit confusing that the way the keys are generated are different when conceptually it's the same idea, right?